Index: screens/Feed/FeedScreen.jsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/screens/Feed/FeedScreen.jsx b/screens/Feed/FeedScreen.jsx
--- a/screens/Feed/FeedScreen.jsx	
+++ b/screens/Feed/FeedScreen.jsx	(date 1684393221852)
@@ -24,6 +24,8 @@
         navigation={navigation}
         back={'Auth'}
         continueTo={'Market'}
+        left={'Back'}
+        right={'Filter'}
       />
       <View style={styles.searchContainer}>
         <TextInput
